src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -19,8 +19,9 @@ describe('<AddRemoveForm />', () => {
   }
   const fetchItemsFailure = jest.fn(() => Promise.reject(new Error('Fetching Failed')))
 
-  beforeEach(() => {
-    wrapper = shallow(<AddRemoveForm {...mockProps.terms} />)
+  beforeEach(async () => {
+    wrapper = await shallow(<AddRemoveForm {...mockProps.terms} />)
+    window.localStorage.clear()
```

Comment:
Since our `Settings` is using `localStorage` to save our changes, I had tests failing because in one test I toggled the setting, it got saved in `localStorage`, so the next time I created a new component it inherited the setting. This is a feature of the code, but a bug in our testing. We are now clearing between tests so each test is acting as a fresh start user.